websites/C/Coub/dist/metadata.json

```diff
@@ -40,6 +40,18 @@
 			"title": "Show current playing button",
 			"icon": "fas fa-external-link-alt",
 			"value": true
+		},
+		{
+			"id": "show_externalLargeImage",
+			"title": "Show community/channel images",
```

Comment:
Is it possible to shorten this? Probably won't display too well

Reply:
thats why there is a tooltip?
![image](https://user-images.githubusercontent.com/17952364/182042727-9b8123c2-1def-4576-8640-bd727fde046c.png)

Reply:
Changing the string to "Show Thumbnail" could be an easy solution here...